services/alerts/src/__tests__/integration.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ReactNode } from 'react'
+import { renderHook, act } from '@testing-library/react-hooks'
+import { AlertsProvider, useAlerts, useAlert } from '../index'
+
+const wrapper = ({ children }: { children?: ReactNode }) => (
+    <AlertsProvider>{children}</AlertsProvider>
+)
+
+describe('useAlert and useAlerts used together', () => {
```

Comment:
We could also ensure that ids are incremented when using showing multiple alerts and/or remove an alert between multiple `show` calls

Reply:
Yeah good point, I'll add some tests for that. Especially because I already had a bug with the ids (see 8b34c69). I've also come to realise we should add some tests for supplying function args to `useAlert` as well.